CHANGELOG.md

```diff
@@ -1,19 +1,28 @@
 CHANGELOG
 =============
 
-## 0.7.0 (WIP)
+## 0.8.0 (WIP)
 
 This version corresponds to dev-master and is not tagged yet.
 
+## 0.7.0
+
 ### Bug fix
 
 - [#30](https://github.com/satooshi/php-coveralls/issues/30) Fix bug: Guzzle\Common\Exception\RuntimeException occur without response body
+- [#41](https://github.com/satooshi/php-coveralls/issues/41) CloverXmlCoverageCollector could not handle root directory
+- [#114](https://github.com/satooshi/php-coveralls/pull/114) Fix PHP 5.3.3, Fix HHVM on Travis, boost Travis configuration, enhance PHP CS Fixer usage
```

Comment:
It's fun to see 114 in bugfix section, in misc section (add 7.0), but not in enhancement section (well, boost Travis config isn't a bugfix, right?) ;P